docs/en/tutorials/dataset_types.md

```diff
@@ -66,12 +36,98 @@ data = dict(
+User can tune the searching scheme by passing `min_score_thr`, `max_score_thr` and `step` into the evaluation hook in the config.
+
+For example, with the following configuration, you can evaluate the model's output on a list of boundary score thresholds [0.1, 0.2, 0.3, 0.4, 0.5] and get the best score from them **during training**.
+
+```python
+evaluation = dict(
+    interval=100,
+    metric='hmean-iou',
+    min_score_thr=0.1,
+    max_score_thr=0.5,
+    step=0.1)
+```
+
+**During testing**, you can change these parameter values by appending them to `--eval-options`.
+
+```bash
+python tools/test.py configs/textdet/dbnet/dbnet_r18_fpnc_1200e_icdar2015.py db_r18.pth --eval hmean-iou --eval-options min_score_thr=0.1 max_score_thr=0.6 step=0.1
+```
+
+Check out our [API doc](https://mmocr.readthedocs.io/en/latest/api.html#mmocr.core.evaluation.eval_hmean) for further explanations on these parameters.
 
 ### TextDetDataset
 
 *Dataset with annotation file in line-json txt format*
 
+We have designed new types of dataset consisting of **loader** , **backend**, and **parser** to load and parse different types of annotation files.
+- **loader**: Load the annotation file. We now have a unified loader, `AnnFileLoader`, which can use different `backend` to load annotation from txt. The original `HardDiskLoader` and `LmdbLoader` will be deprecated.
+- **backend**: Load annotation from different format and backend.
+  - `LmdbAnnFileBackend`: Load annotation from lmdb dataset.
+  - `HardDiskAnnFileBackend`: Load annotation file with raw hard disks storage backend. The annotation format can be either txt or lmdb.
+  - `PetrelAnnFileBackend`: Load annotation file with petrel storage backend. The annotation format can be either txt or lmdb.
+  - `HTTPAnnFileBackend`: Load annotation file with http storage backend. The annotation format can be either txt or lmdb.
+- **parser**: Parse the annotation file line-by-line and return with `dict` format. There are two types of parser, `LineStrParser` and `LineJsonParser`.
+  - `LineStrParser`: Parse one line in ann file while treating it as a string and separating it to several parts by a `separator`. It can be used on tasks with simple annotation files such as text recognition where each line of the annotation files contains the `filename` and `label` attribute only.
+  - `LineJsonParser`: Parse one line in ann file while treating it as a json-string and using `json.loads` to convert it to `dict`. It can be used on tasks with complex annotation files such as text detection where each line of the annotation files contains multiple attributes (e.g. `filename`, `height`, `width`, `box`, `segmentation`, `iscrowd`, `category_id`, etc.).
+
+#### Example Configuration
+
 ```python
```

Comment:
The example configuration still uses the deprecated `HardDiskLoader`. It might be better to update it to `AnnFileLoader`.